examples/AzureSDKDemoSwift/Source/Common.swift

```diff
@@ -89,7 +89,7 @@ struct AppState {
     static func blobClient(withDelegate delegate: StorageBlobClientDelegate? = nil) throws -> StorageBlobClient {
         if AppState.internalBlobClient == nil {
             guard let application = AppState.application else {
-                let error = AzureError.general("Application is not initialized. Unable to create Blob Storage Client.")
+                let error = AzureError.sdk("Application is not initialized. Unable to create Blob Storage Client.")
```

Comment:
I would suggest that consuming applications should not create instances of AzureError at all. Obviously we can't prevent that since there's no "group internal" concept in Swift, but we should maybe document it in the API doc comments for AzureError and its cases.

Reply:
I replaced it with `fatalError()`